doc/recipe/preprocessor.rst

```diff
@@ -180,22 +180,80 @@ To get an overview on data fixes and how to implement new ones, please go to
 
 Fx variables as cell measures or ancillary variables
 ====================================================
-The following preprocessor may require the use of ``fx_variables`` 
-to be able to perform the computations:
+The following preprocessors may require the use of ``fx_variables`` to be able
+to perform the computations:
+
+============================================================== =====================
+Preprocessor                                                   Default fx variables
+============================================================== =====================
+:ref:`area_statistics<area_statistics>`                        ``areacella``, ``areacello``
+:ref:`mask_landsea<land/sea/ice masking>`                      ``sftlf``, ``sftof``
+:ref:`mask_landseaice<ice masking>`                            ``sftgif``
+:ref:`volume_statistics<volume_statistics>`                    ``volcello``
+:ref:`weighting_landsea_fraction<land/sea fraction weighting>` ``sftlf``, ``sftof``
+============================================================== =====================
+
+If no ``fx_variables`` are specified for these preprocessors, the fx variables
+in the second column are used. If given, the ``fx_variables`` argument
+specifies the fx variables that the user wishes to input to the corresponding
+preprocessor function. The user may specify these by simply adding the names of
+the variables, e.g.,
 
-    - ``area_statistics``
-    - ``mask_landsea``
-    - ``mask_landseaice``
-    - ``volume_statistics``
-    - ``weighting_landsea_fraction``
+.. code-block:: yaml
+
+    fx_variables:
+      areacello:
+      volcello:
+
+or by explicitly adding variable parameters (the key-value pair may be as
+specific as a CMOR variable can permit):
```

Comment:
I don't understand what this sentence means. Could you elaborate?

Reply:
I'll try to rephrase (just copy-pasted this code from the old version).